CHMeetupApp/Sources/Common/Helpers/PushHandler/PushHandler.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 typealias PushSequence = [AnyHashable : Any]
 
-final class PushHandler {
+final class PushHandler{
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ichina/application/blob/d37f3671da5fb1cc47fb978fadbf5ff71a5252b6/CHMeetupApp/Sources/Common/Helpers/PushHandler/PushHandler.swift#L13"></span>Opening braces should be preceded by a single space and on the same line as the declaration.</td>
    </tr>
  </tbody>
</table>